src/Components/TodoList.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+type Props = {
+  visibleTodos: Todo[],
+  deleteTodo:(value:number) => Promise<void>;
+  tempTodo:Todo | null;
+  loading:boolean;
+  idToDelete:number[];
+};
+
+export const TodoList: React.FC<Props> = ({
+  visibleTodos,
+  deleteTodo,
+  tempTodo,
+  loading,
+  idToDelete,
+}) => {
+  return (
+    <section className="todoapp__main">
+      <TransitionGroup>
+        {visibleTodos.map(todo => (
+          <CSSTransition
+            key={todo.id}
+            timeout={300}
+            classNames="item"
+          >
```

Comment:
I think this timeout should be **3000ms** as it should be by the task description and not **300ms**